compat/mingw.c

```diff
@@ -312,7 +313,10 @@ int mingw_core_config(const char *var, const char *value, void *cb)
 		core_fscache = git_config_bool(var, value);
 		return 0;
 	}
-
```

Comment:
Let's not remove the empty line here, which is used for structuring. Instead, enclose the new conditional with empty lines, too.